test/filters/test_sepia.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+import torch
+from torch.autograd import gradcheck
+
+import kornia
+import kornia.testing as utils
+from kornia.testing import assert_close
+
+
+@pytest.mark.parametrize(
+    "batch_shape,expected",
+    [
+        ((2, 3, 1, 1), [[[[1.0]], [[0.8905]], [[0.6936]]], [[[1.0]], [[0.8905]], [[0.6936]]]]),
+        ((3, 1, 1), [[[1.0]], [[0.8905]], [[0.6936]]]),
+    ],
+)
+def test_sepia_calc(batch_shape, expected, device, dtype):
```

Comment:
why outside of the `TestSepia` ?

Reply:
Fix at https://github.com/kornia/kornia/pull/1947/commits/88dc8d1f280f49b8d87d8e96acb8581383aae7d4